bin/eks-create-sc.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+# Copyright (c) 2016-2017 ForgeRock AS. Use of this source code is subject to the
```

Comment:
Could we have copyright messages for all our scripts?
When changing existing scripts, the end date should be 2018.
New scripts should be Copyright (c) 2018 ForgeRock AS.

Reply:
Yes i will add them in the next PR.